### DISCUSSION TITLE
can you run multiple instances on Windows?

### DISCUSSION THREAD

**Discussion by MrAdamo** [ctubio/Krypto-trading-bot#1053] [Q&A] [answered]
Hello,
I followed the instructions on the manual for running multiple instances and after creating and editing multiple `*.sh` files I tried to run the command `make startall` but I get this error:

> $ make startall
ls -1 *.sh | cut -d / -f2 | cut -d \* -f1 | grep -v ^_ | xargs -I % sh -c 'sleep 2;make K=% start -s'
/bin/sh: line 1: screen: command not found
/bin/sh: line 3: screen: command not found
make[1]: *** [Makefile:268: start] Error 127
/bin/sh: line 1: screen: command not found
/bin/sh: line 3: screen: command not found
make[1]: *** [Makefile:268: start] Error 127
/bin/sh: line 1: screen: command not found
/bin/sh: line 3: screen: command not found
make[1]: *** [Makefile:268: start] Error 127
make: *** [Makefile:255: startall] Error 123

Is it possible to run multiple instances in one MSYS2 Terminal or do I need to run one terminal for every single `*.sh` config file?

**Comment by ctubio** [comment, marked as answer]
:koala: im sorry i couldnt find any `screen` package on MSYS2, but the great <kbd>tmux</kbd> should do the same work.

you can install it with  `pacman -S tmux` and the manual is at https://man7.org/linux/man-pages/man1/tmux.1.html

(Makefile targets using `screen` command may need to be removed/replaced on winy yea; i didnt found forces to do it as of today :P)

otherwise multiple terminals to run the bot in the foreground (instead of in the background) may be the only alternative solution yea
